.github/workflows/build-and-test.yml

```diff
@@ -16,14 +15,31 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout 
-      uses: actions/checkout@v2
+      uses: actions/checkout@v4
       timeout-minutes: 1
     - name: Touch .env  
       run: touch .env
       timeout-minutes: 1
     - name: Prepare
-      run: docker compose build test
+      run: echo lol # docker compose build test
       timeout-minutes: 1
     - name: Test
-      run: docker compose run test
+      run: echo whut # docker compose run test
```

Comment:
We might want to switch these back at some point (although I appreciate it makes life much nicer when testing the deployment!)

Reply:
whoops, yes. fixed.